Dockerfile

```diff
@@ -1,13 +1,33 @@
-FROM node:18.17.0
+# Build
+FROM node:18.17.0-slim as build
 WORKDIR /app
 COPY . .
 
-ARG packageVersion=0.7.6
-ENV tarballFileName="gomboc-ai-cli-$packageVersion.tgz"
-ENV tarFileName="gomboc-ai-cli-$packageVersion.tar"
-
 RUN npm install
-RUN npm run build
-RUN npm pack
-RUN gzip -d $tarballFileName
-RUN npm install -g $tarFileName
\ No newline at end of file
+RUN npm run build 
+
+# Prod modules
+FROM node:18.17.0-slim as prod-modules
+WORKDIR /app
+COPY --from=build /app ./
```

Comment:
Does `build` do something to `/app` that needs to be copied?  If so you should explain it in a comment.  If not, then this should be `COPY . .`